dev/bin/knative-installer.sh

```diff
@@ -33,3 +52,15 @@ kubectl apply -f https://github.com/knative-sandbox/eventing-kafka-broker/releas
 
 header_text "Waiting for Knative Eventing Kafka to become ready"
 kubectl wait deployment --all --timeout=900s --for=condition=Available -n knative-eventing
+
+header_text "Initializing Knative Eventing in-memory channel"
+kubectl apply -f https://github.com/knative/eventing/releases/download/knative-v1.5.6/in-memory-channel.yaml
```

Comment:
This was used only in the experiment, I don't think we need it in the cluster

Reply:
removed